src/unix/linux-core.c

```diff
@@ -74,7 +74,7 @@ static void read_speeds(unsigned int numcpus, uv_cpu_info_t* ci);
 static unsigned long read_cpufreq(unsigned int cpunum);
 
 
-int uv__platform_loop_init(uv_loop_t* loop) {
+int uv__epoll_init(uv_loop_t* loop, int inotify) {
```

Comment:
It only *conditionally* does anything with inotify. It primarily calls `uv_epoll_create`.